lib/generators/solidus_afterpay/install/templates/initializer.rb

```diff
@@ -2,4 +2,10 @@
 
 SolidusAfterpay.configure do |config|
   config.use_solidus_api = false
+
+  # There is a cache for retrieving the minimum amount, maximum amount and the currency from Afterpay
+  # the default value us set to 1.day, this is also recommended by Afterpay itself. If you prefer a shorter
+  # or longer time simply uncomment config.cache_expiry and add the preferred value.
+
```

Comment:
Remove this blank line, please since the comment above is related to the comment property below.